frontend/Components/Header.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import {
+	Link,
+} from 'react-router-dom';
+export interface HeaderParams{
```

Comment:
It's [`Props` not `Params`](https://reactjs.org/docs/components-and-props.html) :)